submission/submitter/src/test/kotlin/ac/uk/ebi/biostd/submission/service/AccNoServiceTest.kt

```diff
@@ -29,24 +29,38 @@ private const val PROJECT_PATTERN = "!{ABC-}"
 @ExtendWith(MockKExtension::class)
 class AccNoServiceTest(
     @MockK private val service: PersistenceService,
-    @MockK private val privilegesService: IUserPrivilegesService
+    @MockK private val privilegesService: IUserPrivilegesService,
 ) {
     private val accNoPatternUtil: AccNoPatternUtil = AccNoPatternUtil()
-    private val testInstance = AccNoService(service, accNoPatternUtil, privilegesService)
+    private val testInstance = AccNoService(service, accNoPatternUtil, privilegesService, subBasePath = null)
 
     @AfterEach
     fun afterEach() = clearAllMocks()
 
-    @ParameterizedTest(name = "prefix is {0} and numeric value is {1}")
+    @ParameterizedTest(name = "prefix is {0}, numeric value is {1} and submission base path is {2}")
     @CsvSource(
-        "S-DIXA-AN-002, S-DIXA-AN-/002/S-DIXA-AN-002",
-        "S-BSST11, S-BSST/011/S-BSST11",
-        "S-DIXA-011, S-DIXA-/011/S-DIXA-011",
-        "1-AAA, 1-AAA/000/1-AAA",
-        "S-SCDT-EMBOJ-2019-103549, S-SCDT-EMBOJ-2019-/549/S-SCDT-EMBOJ-2019-103549",
-        "S-SCDT-EMBOR-2017-44445V1, S-SCDT-EMBOR-2017-44445V/001/S-SCDT-EMBOR-2017-44445V1"
+        "S-DIXA-AN-002, S-DIXA-AN-/002/S-DIXA-AN-002, null",
+        "S-BSST11, S-BSST/011/S-BSST11, null",
+        "S-DIXA-011, S-DIXA-/011/S-DIXA-011, null",
+        "1-AAA, 1-AAA/000/1-AAA, null",
+        "S-SCDT-EMBOJ-2019-103549, S-SCDT-EMBOJ-2019-/549/S-SCDT-EMBOJ-2019-103549, null",
+        "S-SCDT-EMBOR-2017-44445V1, S-SCDT-EMBOR-2017-44445V/001/S-SCDT-EMBOR-2017-44445V1, null",
+        "S-DIXA-AN-002, basePath/S-DIXA-AN-/002/S-DIXA-AN-002, basePath",
+        "S-BSST11, basePath/S-BSST/011/S-BSST11, basePath",
+        "S-DIXA-011, basePath/S-DIXA-/011/S-DIXA-011, basePath",
+        "1-AAA, basePath/1-AAA/000/1-AAA, basePath",
+        "S-SCDT-EMBOJ-2019-103549, basePath/S-SCDT-EMBOJ-2019-/549/S-SCDT-EMBOJ-2019-103549, basePath",
+        "S-SCDT-EMBOR-2017-44445V1, basePath/S-SCDT-EMBOR-2017-44445V/001/S-SCDT-EMBOR-2017-44445V1, basePath",
+        "S-DIXA-AN-002, basePath/S-DIXA-AN-/002/S-DIXA-AN-002, basePath/",
+        "S-BSST11, basePath/S-BSST/011/S-BSST11, basePath/",
+        "S-DIXA-011, basePath/S-DIXA-/011/S-DIXA-011, basePath/",
+        "1-AAA, basePath/1-AAA/000/1-AAA, basePath/",
+        "S-SCDT-EMBOJ-2019-103549, basePath/S-SCDT-EMBOJ-2019-/549/S-SCDT-EMBOJ-2019-103549, basePath/",
+        "S-SCDT-EMBOR-2017-44445V1, basePath/S-SCDT-EMBOR-2017-44445V/001/S-SCDT-EMBOR-2017-44445V1, basePath/",
```

Comment:
this number of permutations is too hard to mantain, what each case is is not clear.  Lets add only two more cases to original one.